ansible/roles/gcc/tasks/main.yml

```diff
@@ -2,225 +2,258 @@
 
 #
 # installs gcc 4.9 @ /data/gcc-4.9 on rhel7.x
+# and      gcc 6.3 @ /data/gcc-6.3
 #
 
-- stat: path=/data/gcc-4.9/bin/gcc-4.9
-  register: build_gcc_49
+
+## gmp-6.1
+
+- stat: path=/data/gmp-6.1/include/gmp.h
+  register: build_gmp
   tags: gcc49
 
 - name: Clean path
   file:
     state: absent
     path: /data/tmp/
-  when: build_gcc_49.stat.exists == False
+  when: build_gmp.stat.exists == False
```

Comment:
Should the tags: gcc49 lines which are not specific to 49 like this one be updated to something more generic like `tags:gcc`

Reply:
No idea. The tags have no purpose I could discern.

Reply:
I think they are just a tag to show related items.  In that context I think we could just change them from gcc49 to gcc as those steps are now not specific to the later gcc49 build steps.